client/editor/editor.go

```diff
@@ -41,13 +40,39 @@ type Editor struct {
 	// StatusMsg represents the text displayed in the status bar.
 	StatusMsg string
 
+	// Users is an array of the names of all users connected to the server.
+	Users []string
+
+	// ScrollEnabled determines whether or not the user can scroll past the initial editor window. It is set by the EditorConfig
 	ScrollEnabled bool
+
+	// IsConnected shows whether the editor is currently connected to the server.
+	IsConnected bool
+
+	// A channel for sending status messages
+	StatusChan chan string
+}
+
+var userColors = []termbox.Attribute{
+	termbox.ColorGreen,
+	termbox.ColorYellow,
+	termbox.ColorBlue,
+	termbox.ColorMagenta,
+	termbox.ColorCyan,
+	termbox.ColorLightYellow,
+	termbox.ColorLightMagenta,
+	termbox.ColorLightGreen,
+	termbox.ColorLightRed,
+	termbox.ColorRed,
 }
 
 // NewEditor returns a new instance of the editor.
 func NewEditor(conf EditorConfig) *Editor {
+	statusChan := make(chan string, 20)
```

Comment:
Is there a specific reason for choosing `20` as the capacity? Perhaps it is because status updates are rare events and 20 is an adequate number to handle them?

Reply:
It was an arbitrary choice, but that was my thinking, yeah. Though I don't really see any reason we shouldn't make it bigger. I think we basically never want the channel to block. Maybe a capacity of something like 100 would be more appropriate?